### PR TITLE
Bank organizer

### PR DESCRIPTION
**Pull request by Hubcapp**
https://www.youtube.com/watch?v=6v8X_vFkLPs

### CODE REVIEW

**Comment by Hubcapp** [comment]
Resolves #74, #76, #77 and has the fps limit code from #75 as well.

**Comment by lmsv-mx123** [comment]
Looks good.
Maybe in future expansion there could be sort of free/members (Maybe can be in one of the places somehow 🤔 ). The other expansion commented was the showing Enter search keyword overlay, I'll help you on that one so it can be integrated

File: src/Game/Bank.java [hunk 1]
```diff
@@ -18,191 +18,1056 @@
...
+    int value = 0;
+    for (int idx = 0; idx < Client.new_count_items_bank; idx++) {
+      int itemId = Client.new_bank_items[idx];
+      if (itemId > 1289) {
+        Logger.Info("itemId " + itemId + " at position " + idx + " @|red fwehhhh|@");
+      } else {
+        value += BankSorters.itemValues[itemId] * Client.new_bank_items_count[idx];
+      }
+    }
+    bankValue = String.format("Bank Value: (%d gp)", value);
```
Comment (lmsv-mx123) [inline src/Game/Bank.java:458]: This method is very sweet. However, that the way it operates on shown bank items for calculation, maybe should be set to display if filtered to be "Bank Value ⧩: (%d gp)" or "Filtered Bank: (%d gp)" or "Filtered Value: (%d gp)"

**Review by lmsv-mx123 — APPROVED** [review]

File: src/Game/Bank.java (hunk 1 above)
Comment (lmsv-mx123) [reply src/Game/Bank.java:458]: Also maybe there could be a setting if it wants to be displayed as gp / k / M / B and or colorized

Comment (Hubcapp) [reply src/Game/Bank.java:458]: I agree yes. And market value should be a tick box as well.

**Pull request merged**